src/helpers/files.ts

```diff
@@ -335,7 +335,15 @@ export function getBlocklist(): string[] {
   return [...manualBlocklist(), ...globBlocklist()].map(globstar)
 }
 
-export function cleanCoverageFilePaths(projectRoot: string, paths: string[], ignoreGlobs: string[]): string[] {
+export function filterFilesAgainstBlockList(paths: string[], ignoreGlobs: string[]): string[] {
+  const ignoredFiles = micromatch(paths, ignoreGlobs)
+
+  return paths.filter(path => {
+    return !ignoredFiles.includes(path)
+  })
```

Comment:
```suggestion
  return micromatch.not(paths, ignoreGlobs, {
    contains: true,
  })
```

Disregard the line with `contains` if the contents of `ignoreGlobs` are already globstar'd.